src/languages/English.json

```diff
@@ -31,7 +31,10 @@
   "game_files_no_cached_files": "There are no cached files for the mod!\n\nThe latest version of the ModFiles folder will be downloaded...",
   "workshop_menu_info": "The Workshop Menu is for getting the change level\ncommand to change to a certain workshop map.\n\nMake sure you have the workshop map downloaded\nfirst before running the command in the Portal 2 console!",
   "workshop_link": "Enter a Steam workshop map link:",
-  "workshop_map_not_found": "Could not find workshop map!\nMake sure you are subscribed\nto the map and have played it\nat least once.",
+  "workshop_map_not_found": "Nie znaleziono mapy",
+  "workshop_sentence0_sure-you-are-sub": "Upewnij się że zasubskrybowałeś",
+  "workshop_sentence0_to-map-and-play-it": "mapę i zagrałeś w nią",
+  "workshop_sentence0_least-once": "przynajmniej jeden raz.",
```

Comment:
uhm... i think you mistook the english file for the polish file

Reply:
Yeah, no clue how I made that mistake! I fixed it.